R/moduleInstall.R

```diff
@@ -156,8 +156,11 @@ installJaspModuleFromDescription <- function(modulePkg, libPathsToUse, moduleLib
 
   # TODO: this is not very efficient because renv::install looks up the remotes on github...
   # there is a better way but it requires us to mess with renv's internals or to be more explicit about pkgs
-  renv::hydrate(library = moduleLibrary, project = modulePkg)
-  renv::install(project = modulePkg, library = moduleLibrary, prompt = prompt)
+  # R gurus: the following if-else can be prettier no doubt...
+  if(!is.null(frameworkLibrary))  { sources <- c(moduleLibrary, frameworkLibrary)
+  } else                          { sources <- c(moduleLibrary)                   }
```

Comment:
so I am not actually sure whether you need this if-else at all, as

```r
c("something", NULL)
```

returns

```r
> "something"
```

Reply:
that would def make it much cleaner